mysql-test/suite/binlog_encryption/t/binlog_rotate_key_check_privileges.test

```diff
@@ -0,0 +1,15 @@
+--source include/not_embedded.inc
```

Comment:
Test applies to system keys in general, maybe rename accordingly and move to main suite

Reply:
Done